app/ide-desktop/lib/content/package.json

```diff
@@ -17,7 +17,7 @@
   "scripts": {
     "typecheck": "npx tsc --noEmit",
     "lint": "npx --yes eslint src",
-    "build": "ts-node bundle.ts",
+    "build": "(cd ../dashboard/src/authentication && npx tailwindcss -i src/styles/index.css -o ../../../content/src/dashboard.css -c tailwind.config.ts --minify) && ts-node bundle.ts",
```

Comment:
1. The build time increase is minor (<1 second). But you are correct, best not to have it in the first place. I'll try to add it to the `ts-node bundle.ts` script, maybe? We don't want Tailwind built every time the rust code is built, but it needs to be re-built whenever the JSX in the React files changes, which is pretty much on every build of `lib/dashboard`.
2. Yes, they are hard to maintain but I'm not sure how to reference CSS files across javascript packages without them, since we can't just export the styles.
3. Does not pass the windows CI, but I have an idea about how to fix that.

Reply:
1. Add the same command to the watch 
2. Remove pre-generated tailwind.css from the repository, add on fly generated fly to content/dist/ and add to .gittignore
3. Add fixme for refactor